docs/_chapters/650-windows.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Tips for Windows users
+layout: default
+author: Fr Jeremy Krieg
+---
+
+Bnd and Bndtools runs on any system that can run Eclipse. However, every operating system is different and has its own quirks. Windows in particular has a few that impact developers using Bnd. While the Bnd experience may never be as smooth on Windows as it is on iOS or Linux, there are a few tweaks that make it a lot smoother than what comes out of the box.
+
+It is possible that some of the tips listed here will also help for Eclipse development work in general, or perhaps any development work on Windows. 
+
+## Key problems on Windows 
+
+Windows has two key architectural differences to *nix-based OSes that really impact development work in Java:
+
+### File locking
+
+Most \*nix systems use advisory file locking, rather than mandatory locking. Windows uses mandatory locking rather than advisory. Most Java programs are developed on \*nix and tend to assume \*nix locking semantics. This means that they are not as careful at cleaning up after themselves and ensuring that file locks are released.
+
+In particular, there are problems when an application exits using `System.exit()` - the normal way for Javva applications to clean up is to release a lock is in a `finally` clause. However, when an application calls `System.exit()`, these normal channels are bypassed, which means the JVM never gets a chance to release the locks and hence can't delete the locked files.
+
+### Real-time protection
+
+From version 10, Windows' built-in anti-malware program Windows Defender has become very tightly integrated with the OS - seemingly, it is written into the Windows kernel API IO functions themselves, which means it's nearly impossible to bypass. This makes it extremely well suited to catching malware in the act. Unfortunately, it means it slows down every file write. When you are writing lots of small files, this delay is significant.
+
+## Workarounds and solutions
+
+### Use Bnd 5+
+
+Prior to Bnd 5.0, there was a bug that caused the launcher to pause for several seconds if you had `[-runtrace](/instructions/runtrace.html)` enabled. This bug has been fixed as of 5.0 (currently in pre-release at the time of writing), so it is worthwhile upgrading to use 5.0 if you are able.
```

Comment:
You can't put the back tiks around the the whole link. Just put them inside the square braces around `-runtrace`. Otherwise the link is not rendered as a link.

Reply:
Fixed.